human_aware_rl/ppo/ppo_rllib_from_params_client.py

```diff
@@ -0,0 +1,410 @@
+# All imports except rllib
```

Comment:
Would it be possible to coalesce this file with the `ppo_rllib_client.py` file? As far as I can tell the only difference is with the `environment_params` parameter, albeit the parameter setup for this file seems a lot more complicated because it relies on a local schedule function. If it's not possible to to merge the two, could you possibly import the `ex` from `ppo_rllib_client` and override only the environment_params config, or would that mess with sacred command line parsing?

I only ask because it seems like a bad idea for long term maintenance to have so much duplicated code

Reply:
That's a good question. 

I tried to avoid editing ppo_rllib_client because never used it, and I prefer to have the freedom to change it because I feel like this is very project-dependent. And variable-mdp tends to require a different set of default parameter